sdk/eventhub/Azure.Messaging.EventHubs/tests/Snippets/Sample06_IdentityAndSharedAccessCredentialsLiveTests.cs

```diff
@@ -81,7 +81,7 @@ public async Task SharedAccessSignature()
 
             #region Snippet:EventHubs_Sample06_SharedAccessSignature
 
-            //@@ var credential = new AzureSasCredential("<< SHARED ACCESS KEY STRING >>");
+            var credential = new AzureSasCredential("<< SHARED ACCESS KEY STRING >>");
```

Comment:
Orthogonal, but I forgot the SAS credential doesn't do eager validation; we can use this form.